### DISCUSSION TITLE
Proyecto Final Grupo Susana

### DISCUSSION THREAD

**Discussion by DavidGarciaCosta** [mmasias/24-25-IdSw1#132] [General]
Buenos días, escribo en nombre del grupo de Susana Gutiérrez.
Formado por Álvaro Lostal,Daniel Puente, Raúl Pelayo, Oscar Santamaría y David García.
Nos gustaría presentar el trabajo el día 16 de Enero de 2025. 
Muchas gracias

**Comment by mmasias** [comment]
> Buenos días, escribo en nombre del grupo de Susana Gutiérrez. Formado por Álvaro Lostal,Daniel Puente, Raúl Pelayo, Oscar Santamaría y David García. Nos gustaría presentar el trabajo el día 16 de Enero de 2025. Muchas gracias

Por favor, crear una entrada en Google Calendar en el que estén todos los integrantes del grupo y yo.
